src/jit/codegenarm.cpp

```diff
@@ -78,7 +78,7 @@ void CodeGen::instGen_Set_Reg_To_Imm(emitAttr size, regNumber reg, ssize_t imm,
 
     if (EA_IS_RELOC(size))
     {
-        genMov32RelocatableImmediate(size, imm, reg);
+        genMov32RelocatableImmediate(size, (BYTE*)imm, reg);
```

Comment:
Can you add a GitHub issue to track cleaning this up so we don't need to do this cast? I believe we determined that all the callers of instGen_Set_Reg_To_Imm that know it's relocatable already cast a pointer to this `ssize_t` imm.

Reply:
Thank for suggestion! Tracked as #19091